### ISSUE TITLE
page 40 formula2.46

### ISSUE THREAD

**Issue by qile2000** [probml/pml-book#235]
E[E[X|Y]] is represented as E[E[X|Y]|Y]. Should E[E[X|Y]|Y] be replaced by E_Y[E[X|Y]]? (here X denotes corresponding variable)

**Comment by pglpm** [comment]
Indeed, I think that "E[ E[_X_|_Y_] | _Y_]" in that formula (where "E[...]" is shortened as "_s_...") is meaningless. After calculating  E[_X_|_Y_] the only remaining unknown quantity is _Y_; so if we now ask for E[?|_Y_], what's in the argument of the probability?
Analogously for "E[ E[_X_^2|_Y_] | _Y_]".

**Comment by liangwu82** [comment]
Conditioning on Y in 2.46, 2.47 seems unnecessary.
